lnd_test.go

```diff
@@ -3826,6 +3826,7 @@ func testSpiderShortestPath(net *lntest.NetworkHarness, t *harnessTest) {
 	// Initialize seed random in order to generate invoices.
 	prand.Seed(time.Now().UnixNano())
 	var wg sync.WaitGroup
+	var tranwg sync.WaitGroup
```

Comment:
Can you add a comment to explain what these wait groups are doing and how they are relevant below?